docs/computing/alliancecan.md

```diff
@@ -156,6 +171,59 @@ ssh -Y <user>@graham.computecanada.ca
 By default, this brings you to the login node which **is not** a compute node. 
 This should only be used for viewing images.
 
+### Run Jupyter Notebooks in your local VS Code
+To be able to run Jupyter Notebooks in Visual Studio Code from your local machine, use the following instructions:
+1. Follow the instructions above to set the passwordless SSH.
+2. Open your local VS Code and install the `Remote - SSH` extension.
+3. Hit `Ctrl+Shift+p` and type `Remote-SSH: Open SSH configuration file...`. Select the file in your home directory and set it to:
+    ```
+    Host graham.sharcnet.ca
+        HostName graham.sharcnet.ca
+        User <username>
+        IdentityFile ~/.ssh/<file>
+    ```
+    Replace `<username>` with your corresponding Graham username and `<file>` with the file generated when setting the passwordless SSH connection.
+4. Before connecting to the server, go to the settings of the Remote-SSH extension and set the Connect Timeout value to a higher value than the default, for example, 10000 (since Graham usually responds slowly).
+5. Use `Ctrl+Shift+p` again, type `Remote-SSH: Connect to Host...` and use the previous config file. This will enable the SSH connection between VS Code and Graham. You should now be able to see your Graham folders in VS Code.
+6. In a terminal, ssh to Graham. 
+    1. Create a virtual environment using `kpy create <name>` (from [kslurm](https://github.com/pvandyken/kslurm) tool). 
+    2. Load the venv and install `jupyterlab` plus any other packages you need.
+    3. Save the env (`kpy save <name of your venv>`) and deactivate the venv (`deactivate`). 
+    4. Run `kjupyter` with any required specifications and using the created venv (refer to the instructions [here](https://github.com/pvandyken/kslurm)).
+7. Return to the VS Code window from step 5 and open the Jupyter file that you want to run.
+8. Use `Ctrl+Shift+p` and type `Jupyter: Specify Jupyter Server for Connections`. Paste the URL given by kjupyter for the server connection.
+
+>
+- If kjupyter command doesn't work after step 6, try installing `jupyterlab` in your local Python in Graham (without any venv activated, run `pip install jupyterlab`).
+- Running Jupyter Notebooks through SSH Tunneling is usually faster. When running `kjupyter`, this option is provided.  
+{: .note}
+
+### Visualization of Compute Nodes
```

Comment:
A minor point, but might good to combine the two visualization sections with the two options:

1. X forwarding
2. VNC

Reply:
Done, merged them!